.github/ci/unbottled_dependencies.bash

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
If this script has to be run on macOS it may make sense to write it not to use bash, given that the version of Bash available without homebrew is heinously out-of-date. (or it's been removed, I can't remember)

Reply:
I'm open to that; do you recommend `zsh`?

Reply:
switched to `/bin/sh` in https://github.com/osrf/homebrew-simulation/pull/2610/commits/d3e7b1df4c66bfa948b89204ebf8a66eaecbd518